packages/web/src/ui/flow/design.js

```diff
@@ -423,7 +423,7 @@ const actions = ({ sources }) => {
   const setDesignContent$ = most.mergeArray(
     Object.values(sources).filter((x) => x !== undefined && 'source' in x)
   )
-    .delay(1000)// FIXME delay is a hack
+    //.delay(1000)// FIXME delay is a hack
```

Comment:
Please remove line 426. The comment FIXME is not necessary.

Reply:
ok